blog-server-services/src/impls/rbatis_post_service.rs

```diff
@@ -65,12 +65,12 @@ impl Post {
             author.slug AS author_slug, \
             author.first_name AS author_first_name, \
             author.last_name AS author_last_name, \
-            GROUP_CONCAT(CONCAT_WS(',', tag.slug, tag.title) SEPARATOR ';') as tags \
+            string_agg(concat_ws(',', tag.title, tag.slug), ';') as tags \
```

Comment:
let's use `concat_ws` lowercase for two other cases.

Reply:
sure! thank for ur attention! =)